src/compiler/utilities.ts

```diff
@@ -1591,6 +1590,11 @@ namespace ts {
         return parameter && parameter.symbol;
     }
 
+    export function getNodeCommentedByJSDocTag(node: JSDocTag): Node {

```

Comment:
we should really make up a name for "the node that this jsdoc is attached to". Commentee? Given that the parameter name and type could be `tag: JSDocTag`, I think `getCommentee` would be a fine name, given a good word for "Commentee".

Reply:
I've been calling it the "host" node in my internal monologue.